SDVModTest/Extensions/CollectionExtensions.cs

```diff
@@ -11,7 +7,7 @@ static class CollectionExtensions
 
         public static TValue SafeGet<Tkey, TValue>(this IDictionary<Tkey, TValue> dictionary, Tkey key, TValue defaultValue = default(TValue))
         {
-            TValue value = defaultValue;
+            var value = defaultValue;

```

Comment:
I like var but I have a rule on its usage. The type must be visible on at least one side of the assignment. I've run into bugs where we thought we were dealing with one type but were actually dealing with something else entirely.